### ISSUE TITLE
Fine tune last x layers of TransformerWordEmbeddings

### ISSUE THREAD

**Issue by Avee-81** [flairNLP/flair#1918]
Hello

I am trying to make use of the Bert model through the TransformerWordEmbeddings class available with Flair.

I wish to know if there is any option available to fine tune just the last "x"(last 1 or 2 or 3) layers of Bert or not? Fine tuning the entire BERT model is turning out to be very time consuming and I don't have that much of GPU resources with me. Can you suggest me a way to just a fine tune the last layers of Bert and freeze the other layers. 

Thanks!

**Comment by helpmefindaname** [comment]
hello

it's not possible to only finetune certain layers, however you can solve memory-problems on the GPU in another way.
in the trainer set the `mini_batch_chunk_size` to a low enough value. 
This calculates the gradient in smaller batches while still using `mini_batch_size` examples for the calculation of the average, so you use less memory for the same result.

**Comment by djstrong** [comment]
Generally, it is possible to fine-tune certain layers, but I am not sure if it saves so much memory and time.
